ChakraCore Samples/Hello World/Linux_OSX/Makefile

```diff
@@ -9,13 +9,16 @@ CC=g++
 LIBRARY_PATH=../BuildLinux/$(BUILD_TYPE)/lib
 LDIR=$(LIBRARY_PATH)/../pal/src/libChakra.Pal.a \
 	$(LIBRARY_PATH)/Common/Core/libChakra.Common.Core.a \
-	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a
+	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a \
+	/usr/local/opt/icu4c/lib/libicudata.a \
+	/usr/local/opt/icu4c/lib/libicuuc.a \
+	/usr/local/opt/icu4c/lib/libicui18n.a 
```

Comment:
The path of the lib icu4c depends on the way you installed the library, in my case I used brew install icu4c, and the library is on /usr/local/Cellar/icu4c/54.1/lib, probably we need to consider to include a variable LIBRARY_ICU4C_PATH=/usr/local/Cellar/icu4c/54.1/lib